package.json

```
@@ -7,7 +7,7 @@
     "build": "webpack -p",
     "dev": "webpack-dev-server --mode development",
     "lint": "eslint --ext .js src",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "tape ./test/unit/*.js"
```

could we instead do something like

```suggestion
    "test": "npm run lint && npm run test:unit"
    "test:unit": "tape ./test/unit/*.js"
```